src/theme.ts

```diff
@@ -203,12 +203,39 @@ const nord = {
   logoColor: '#d8dee9',
 }
 
+const midnight = {
+  backgroundColor: '#1c1e2d', 
+  textColor: '#d3d7e1', 
+  textColorSecondary: 'rgba(211, 215, 225, 0.7)', 
+  badgeColors: {
+    [Rating.S]: '#3A506B',
+    [Rating.A]: '#4C6A92',
+    [Rating.B]: '#5C7A9D',
+    [Rating.C]: '#3D4C6D',
+    [Rating.D]: '#2B3A4A',
+    [Rating.E]: '#1D2A38' 
+  },
+  badgeTextColors: {
+    [Rating.S]: '#ffffff',
+    [Rating.A]: '#ffffff',
+    [Rating.B]: '#ffffff',
+    [Rating.C]: '#ffffff',
+    [Rating.D]: '#ffffff',
+    [Rating.E]: '#ffffff' 
+  },
+  barBackground: '#2c3e50',
+  barForeground: '#ecf0f1',
+  borderColor: '#34495e',
+  avatarPlaceholderColor: '#7f8c8d',
+  logoColor: '#ecf0f1'
+}
+
 export const preset: Record<string, Theme> = {
   light,
   dark,
   sepia,
   solarizedLight,
   solarizedDark,
   tokyoNight,
-  nord,
```

Comment:
It looks like the Midnight theme was successfully added, but the Nord theme was removed on this line. You may need to add it back.